elastica/dissipation.py

```diff
@@ -144,13 +144,13 @@ def __init__(self, damping_constant, time_step, **kwargs):
             * np.diagonal(self._system.inv_mass_second_moment_of_inertia).T
         )
 
-    def dampen_rates(self, rod: SystemType, time: float):
-        rod.velocity_collection[:] = (
-            rod.velocity_collection * self.translational_damping_coefficient
+    def dampen_rates(self, system: SystemType, time: float):
+        system.velocity_collection[:] = (
+            system.velocity_collection * self.translational_damping_coefficient
         )
 
-        rod.omega_collection[:] = rod.omega_collection * np.power(
-            self.rotational_damping_coefficient, rod.dilatation
+        system.omega_collection[:] = system.omega_collection * np.power(
+            self.rotational_damping_coefficient, system.dilatation
```

Comment:
maybe we should revert this back? since currently `dilatation` is not defined for rigid bodies

Reply:
f079abb